shim/shim.c

```diff
@@ -368,7 +368,7 @@ read_IO_message(struct cc_shim *shim, uint64_t *seq, ssize_t *stream_len) {
 			}
 		}
 	}
-	*seq = get_big_endian_64(buf);
+	*seq = get_big_endian_64((uint8_t*)buf);
```

Comment:
but then the return value is used elsewhere in a atoi(buf). I'd argue the cast is probably better there (interpreting part of the byte stream as something else), but it doesn't really matter, let's drop this.